VerySimpleExpressions.lhs

```diff
@@ -766,9 +777,62 @@ parsing code.
 
 == Testing with the examples
 
-TODO: write a little manual tester that accepts a parser and a list of
-examples, and checks they all parse correctly.
-
+> quickcheck :: (Eq a, Show a) => (Parser a -> String -> Either ParseError a) -> (Parser a) -> [(String, Either ParseError a)] -> IO ()
```

Comment:
The quickcheck function is good, but needs a different name since the word 'quickcheck' is already established in the haskell community as a different approach to testing and a library: https://hackage.haskell.org/package/QuickCheck. Can you find a good name which isn't already used by a testing library?